File: 3249/3249.py
# bee 3249 - Death Knight Hero
# Alonso Martins
# 11/09/2023
N = int(input())
vitorias = 0
for i in range(N):
    s = input()
    if "CD" not in s:
        vitorias += 1
print(vitorias)

File: 3249/3249.c
// bee 3249 - Death Knight Hero
// Alonso Martins
// 05/04/2024
#include <stdio.h>
#include <string.h>

int main()
{
    int N, vitorias = 0;
    char k[1001];
    scanf("%d", &N);
    
    while (N--)
    {
        scanf("%s", k);
        if (strstr(k, "CD") == NULL)
            vitorias ++;
    }
    
    printf("%d\n", vitorias);
    
    return 0;
}
